xbmc/utils/LangCodeExpander.cpp

```diff
@@ -1268,198 +1268,199 @@ const std::array<struct LCENTRY, 538> g_iso639_2 = {{
-  { "mn", "mon", NULL, NULL },
-  { "my", "bur", NULL, "mya" },
-  { "na", "nau", NULL, NULL },
-  { "nv", "nav", NULL, NULL },
-  { "nr", "nbl", NULL, NULL },
-  { "nd", "nde", NULL, NULL },
-  { "ng", "ndo", NULL, NULL },
-  { "ne", "nep", NULL, NULL },
-  { "nl", "dut", "nld", "nld" },
-  { "nn", "nno", NULL, NULL },
-  { "nb", "nob", NULL, NULL },
-  { "no", "nor", NULL, NULL },
-  { "ny", "nya", NULL, NULL },
-  { "oc", "oci", NULL, NULL },
-  { "oj", "oji", NULL, NULL },
-  { "or", "ori", NULL, NULL },
-  { "om", "orm", NULL, NULL },
-  { "os", "oss", NULL, NULL },
-  { "pa", "pan", NULL, NULL },
-  { "pi", "pli", NULL, NULL },
-  { "pl", "pol", "plk", NULL },
-  { "pt", "por", "ptg", NULL },
-  { "ps", "pus", NULL, NULL },
-  { "qu", "que", NULL, NULL },
-  { "rm", "roh", NULL, NULL },
-  { "ro", "rum", "ron", "ron" },
-  { "rn", "run", NULL, NULL },
-  { "ru", "rus", NULL, NULL },
-  { "sh", "scr", NULL, NULL },
-  { "sg", "sag", NULL, NULL },
-  { "sa", "san", NULL, NULL },
-  { "si", "sin", NULL, NULL },
-  { "sk", "slo", "sky", "slk" },
-  { "sl", "slv", NULL, NULL },
-  { "se", "sme", NULL, NULL },
-  { "sm", "smo", NULL, NULL },
-  { "sn", "sna", NULL, NULL },
-  { "sd", "snd", NULL, NULL },
-  { "so", "som", NULL, NULL },
-  { "st", "sot", NULL, NULL },
-  { "es", "spa", "esp", NULL },
-  { "sq", "alb", NULL, "sqi" },
-  { "sc", "srd", NULL, NULL },
-  { "sr", "srp", NULL, NULL },
-  { "ss", "ssw", NULL, NULL },
-  { "su", "sun", NULL, NULL },
-  { "sw", "swa", NULL, NULL },
-  { "sv", "swe", "sve", NULL },
-  { "ty", "tah", NULL, NULL },
-  { "ta", "tam", NULL, NULL },
-  { "tt", "tat", NULL, NULL },
-  { "te", "tel", NULL, NULL },
-  { "tg", "tgk", NULL, NULL },
-  { "tl", "tgl", NULL, NULL },
-  { "th", "tha", NULL, NULL },
-  { "ti", "tir", NULL, NULL },
-  { "to", "ton", NULL, NULL },
-  { "tn", "tsn", NULL, NULL },
-  { "ts", "tso", NULL, NULL },
-  { "tk", "tuk", NULL, NULL },
-  { "tr", "tur", "trk", NULL },
-  { "tw", "twi", NULL, NULL },
-  { "ug", "uig", NULL, NULL },
-  { "uk", "ukr", NULL, NULL },
-  { "ur", "urd", NULL, NULL },
-  { "uz", "uzb", NULL, NULL },
-  { "ve", "ven", NULL, NULL },
-  { "vi", "vie", NULL, NULL },
-  { "vo", "vol", NULL, NULL },
-  { "wa", "wln", NULL, NULL },
-  { "wo", "wol", NULL, NULL },
-  { "xh", "xho", NULL, NULL },
-  { "yi", "yid", NULL, NULL },
-  { "yo", "yor", NULL, NULL },
-  { "za", "zha", NULL, NULL },
-  { "zh", "chi", "zho", "zho" },
-  { "zu", "zul", NULL, NULL },
-  { "zv", "und", NULL, NULL }, // Kodi intern mapping for missing "Undetermined" iso639-1 code
-  { "zx", "zxx", NULL, NULL }, // Kodi intern mapping for missing "No linguistic content" iso639-1 code
-  { "zy", "mis", NULL, NULL }, // Kodi intern mapping for missing "Miscellaneous languages" iso639-1 code
-  { "zz", "mul", NULL, NULL }  // Kodi intern mapping for missing "Multiple languages" iso639-1 code
-} };
+const std::array<ISO639, 189> LanguageCodes = {{
+    {"aa", "aar", NULL, NULL},
```

Comment:
If you make the cosmetics can we change the `NULL` direct to `nullptr` to match also the C++11.

Thanks a lot with this cosmetics, as it reduce clang style problems for future.

Reply:
Forget it for here, is a backport then, leave as is to have both equal :smile:

Reply:
yes, maybe in a follow up. let's first get this thing ready 😉